update.go

```
@@ -30,14 +41,40 @@ type pauseToggleMsg struct {
+}
+
+func (m model) playQueueEntry(index int) (model, tea.Cmd) {
+	if index < 0 || index >= len(m.queue) {
+		return m, nil
+	}
+	m.queueIndex = index
+	m.state = StateLoading
+	m.elapsedSeconds = 0
+
+	return m, tea.Batch(m.loadingSpinner.Tick, m.progress.SetPercent(0), func() tea.Msg {
+		m.player.Stop()
```

golangci-lint (2.12.2)

[error] Error return value of `m.player.Stop` is not checked

(errcheck)